koji/playbooks/add-module-metadata.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: localhost
```

Comment:
This could also be a step in the previous playbook gated by a variable. I, however, chose to start with discrete steps since these are two separate actions.